mailbox/plugin/deleted-messages-vault/src/test/java/org/apache/james/vault/DeletedMessageZipperTest.java

```diff
@@ -88,8 +89,8 @@ void zipShouldPutEntriesToOutputStream() throws Exception {
 
             try (ZipAssert zipAssert = assertThatZip(outputStream)) {
                 zipAssert.containsOnlyEntriesMatching(
-                        hasName(MESSAGE_ID.serialize()).hasStringContent(MESSAGE_CONTENT),
-                        hasName(MESSAGE_ID_2.serialize()).hasStringContent(MESSAGE_CONTENT));
+                        hasName(MESSAGE_ID.serialize() + ZIPPED_FILE_EXTENSION).hasStringContent(MESSAGE_CONTENT),
```

Comment:
Please use raw values in test

BTW we want `xyz.zip` and not `xyzzip` (you miss the `.`)